docker/Dockerfile_stack

```diff
@@ -26,6 +26,9 @@ RUN apt-get update && apt-get install -y \
         libatlas-base-dev \
         zlib1g-dev \
         python2.7-dev \
+        ninja-build \
+        python-sphinx \
+        python-breathe \
```

Comment:
We need specific versions of sphinx and breathe.  We will use pip to install them.